environments/.caas/inventory/group_vars/all/manila.yml

```diff
@@ -8,6 +8,7 @@ caas_manila_home:
 cluster_project_manila_share_name: azimuth-project-share
 caas_manila_project:
   share_name: "{{ cluster_project_manila_share_name }}"
+  share_user: "{{ cluster_project_manila_share_user }}"
```

Comment:
I think we need to default these two to None or similar? Because when we only pass them when cluster_project_manila_share = true.